### DISCUSSION TITLE
"groupadd: Permission denied" while starting Docker-Image

### DISCUSSION THREAD

**Discussion by Tobias-08** [renovatebot/renovate#16329] [Request Help] [answered]
Hi,
I'm using the official Docker image (latest) on Azure Devops Server. While starting the image I'm getting following error:

```
Grant user 'ubuntu' SUDO privilege and allow it run any command without authentication.
groupadd: Permission denied.
groupadd: cannot lock /etc/group; try again later.
```

Any ideas?

Thanks!
Tobias

**Comment by PhilipAbed** [comment]
seems to me like permissions on your Azure Devops Server, how's that related to renovate ?

**Reply by PhilipAbed** [reply]
check which UUID the agent is running could help

**Comment by Tobias-08** [comment, marked as answer]
Thanks! Adding `--user 0:0` seems to be necessary in Azure Devops Server and solved the problem.

**Reply by viceice** [reply]
you should try `1000:0` instead, otherwise renovate will run as root, which can cause other bad side effects.
